bigcache.go

```diff
@@ -22,6 +22,19 @@ type BigCache struct {
 	maxShardSize uint32
 }
 
+// RemoveReason is a value used to signal to the user why a particular key was removed in the OnRemove callback.
+type RemoveReason int32
+
+const (
+	// Expired means the key is past its LifeWindow.
+	Expired = iota
```

Comment:
`Expired RemoveReason = iota`, without an explicit type it will be `int`.
See example https://play.golang.org/p/mZZdMaI92cI

Reply:
Neat. I'll update.